app/frontend/src/features/messages/TabBar.tsx

```diff
@@ -0,0 +1,32 @@
+import { Tab } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import React from "react";
+import { TabList } from "@material-ui/lab";
+
+const useStyles = makeStyles({ root: {} });
+
+export interface TabBarProps<T extends Record<string, string>> {
+  value: keyof T;
+  setValue: (value: keyof T) => void;
+  labels: T;
+}
+
+export default function TabBar<T extends Record<string, string>>({ value, setValue, labels }: TabBarProps<T>) {
+  const handleChange = (event: any, newValue: keyof T) => {
+    setValue(newValue);
+  };
+  const classes = useStyles();
+  return (
+    <TabList
```

Comment:
What is the difference between TabList and Tabs

Reply:
I do not know exactly. I see the children props is of different type and on the demo page TabList is related to accessibility.